rorocloud/client.py

```diff
@@ -144,6 +148,7 @@ def __init__(self, data):
         self.status = data["status"]
         self.start_time = data["start_time"]
         self.end_time = data["end_time"]
+        self.instance_type = data["instance_type"]
```

Comment:
What if the client is used with an older version of server? We should make sure, it doesn't crash in such cases. 

Please make it `data.get("instance_type")` instead of `data["instance_type"]`.